src/main/java/io/lettuce/core/RedisClient.java

```diff
@@ -274,6 +312,10 @@ public static RedisClient create(ClientResources clientResources, RedisURI redis
             writer = new CommandExpiryWriter(writer, getOptions(), getResources());
         }
 
+        if (listeners != null && !listeners.isEmpty()) {
```

Comment:
:+1:

We need to consider all other connect methods, too (Sentinel, Redis Cluster)

Reply:
done